gen3-integration-tests/gen3_ci/jenkins_jobs/check-kube-pod.groovy

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
can we rename this to something like `check-kube-job` or `check-job-success` or something like that? it might be misunderstood as checking a kube pod in general